modules/secure-cloud-run-net/network.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_subnetwork" "vpc_subnetwork" {
```

Comment:
Can we move this into the (forthcoming) composition module to support the greenfield standalone ("click to deploy") use case?  In the brownfield use case, we would need to insert this into one of the 3-network patterns in SFBP.

Reply:
@mitchelljamie the creation of this resource is optional. We have a variable "create_subnet" so the user can choose to create or not. We also have other variable called "subnet_name" that we can pass a name that already exists to be used on the connector!

Do you think this approach solves the issue?

Reply:
Yep I think it does, thanks @Samir-Cit!